zenn/article.go

```diff
@@ -38,14 +40,15 @@ func NewZennArticleFromQiitaArticle(a *qiita.Article) *ZennArticle {
 		topics = append(topics, v.Name)
 	}
 	return &ZennArticle{
-		Title:     a.Title,
-		Emoji:     "😀",
-		Type:      "tech",
-		Topics:    topics,
-		Published: false,
-		Body:      a.Body,
-		ImageURLs: a.ImageURLs,
-		Slug:      slug,
+		Title:       a.Title,
+		Emoji:       "😀",
+		Type:        "tech",
+		Topics:      topics,
+		Published:   false,
+		PublishedAt: a.CreatedAt.Format("2006-01-02 03:04"),
```

Comment:
時刻の指定が12時間表記になってしまってそうです。

```suggestion
		PublishedAt: a.CreatedAt.Format("2006-01-02 15:04"),
```